### PR TITLE
add documentation for cde,humanatlas and dashboards in readme

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,180 +1,170 @@
...
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
 
-- FTU User Interface (FTU)
-  - Production: <https://apps.humanatlas.io/ftu-explorer/>
-  - Staging: <https://hra-ui.netlify.app/apps/ftu-ui/>
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
```
Comment (axdanbol) [inline README.md:21]: Point this to the hra-ui repo

File: README.md [hunk 2]
```diff
@@ -1,180 +1,170 @@
...
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
```
Comment (axdanbol) [inline README.md:22]: Upload the hra-ui logo

File: README.md [hunk 3]
```diff
@@ -1,180 +1,170 @@
...
+</details>
 
-## Change Log
+<!-- ABOUT THE PROJECT -->
 
-See the [ChangeLog](CHANGELOG.md) for the latest developments.
+## About The Project
 
-## Credits
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
```
Comment (axdanbol) [inline README.md:55]: Point this to https://apps.humanatlas.io/ and upload a screenshot. You can probably ask Libby to create a good screenshot of multiple apps in a single image

File: README.md [hunk 4]
```diff
@@ -1,180 +1,170 @@
...
 
-This software is developed by the [Cyberinfrastructure for Network Science Center at Indiana University](http://cns.iu.edu/).
+This repository contains all the Human Reference Atlas (HRA) User Interfaces
```
Comment (axdanbol) [inline README.md:58]: List each app with a short description here and link to the readme of each app

File: README.md [hunk 5]
```diff
@@ -1,180 +1,170 @@
...
-![Indented List](projects/asctb-reporter/src/assets/snippets/il.png)
+```sh
+$ npx nx run cde-ui:serve
+```
 
-Info Sheet
+For more examples, please refer to the documentation for each app
 
-![Info Bottom Sheet](projects/asctb-reporter/src/assets/snippets/bottom.png)
+- [ASCT+B API](./apps/asctb-api/README.md)
```
Comment (axdanbol) [inline README.md:112]: Move app links up to about section

File: README.md [hunk 6]
```diff
@@ -1,180 +1,170 @@
...
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
 
-## Change Log
+<!-- ABOUT THE PROJECT -->
 
-See the [ChangeLog](CHANGELOG.md) for the latest developments.
+## About The Project
```
Comment (axdanbol) [inline README.md:53]: Change to `About HRA UI`

File: README.md [hunk 7]
```diff
@@ -1,180 +1,170 @@
...
-- Export
-- Visualization Functions
-- Visualization Controls
+<!-- USAGE EXAMPLES -->
 
-### Documentation
+## Usage
 
-Click [here](https://ccf-asct-reporter.netlify.app/docs) to view the documentation.
+To serve any app use this command
```
Comment (axdanbol) [inline README.md:98]: Make sure to end all sentences with a period `.`.

**Review by axdanbol — CHANGES_REQUESTED** [review]
Make sure to use complete sentences and end each sentence with a period `.`.

File: README.md (hunk 3 above)
Comment (bhushankhope) [reply README.md]: Updated the URL. Created a new issue for libby to create a image for HRA Apps

**Pull request merged**